benchmarks-mobile/protonano_app/app/src/main/java/io/grpc/benchmarks/ProtobufBenchmarker.java

```diff
@@ -0,0 +1,165 @@
+package io.grpc.benchmarks;
+
+import com.google.protobuf.nano.CodedOutputByteBufferNano;
+import com.google.protobuf.nano.MessageNano;
+
+import org.json.JSONException;
+import org.json.JSONObject;
```

Comment:
Are you sure you want to use these?  Android defaults tend to be /really/ bad.  Real programs use either Jackons or Gson, and serialize the result into an actual java pojo.